tests/integration_tests/data_steward/gcloud/bq_client_test.py

```diff
@@ -20,3 +21,19 @@ def setUpClass(cls):
     def setUp(self):
         self.project_id = app_identity.get_application_id()
         self.bq_client = BigQueryClient(self.project_id)
+        unioned_dataset_id = os.environ.get('UNIONED_DATASET_ID')
```

Comment:
Note for me:
This test is copied from `curation/tests/unit_tests/data_steward/utils/bq_test.py` and updated to fit this new client.

Reply:
`curation/tests/integration_tests/data_steward/utils/bq_test.py` also has something very similar.